### ISSUE TITLE
Barcode contains whitespace when displayValue is false

### ISSUE THREAD

**Issue by angeloashmore** [lindell/JsBarcode#77]
**Issue**: Barcode contains whitespace when displayValue is false.
**Cause**: The barcode width calculation ignores the `displayValue` option.

Some barcode types will render some characters of the `displayValue` before and/or after the barcode (e.g. UPC). When `displayValue` is set to false, the text is not rendered, but the total width remains the same. This results in whitespace before and/or after the barcode.

This can be seen as intended to ensure standardized output widths, but an option to remove all margins would be nice.

**HTML**:
```html
<svg id="upc-a"></svg>
```

**JavaScript** with `displayValue: true`:
```js
JsBarcode("#upc-a", "123456789012", {
  displayValue: true,
  format: "upc",
  background: "#000",
  lineColor: "#fff",
  margin: 0
})
```

**Result**:
<img width="261" alt="screen shot 2016-06-24 at 1 38 21 pm" src="https://cloud.githubusercontent.com/assets/8601064/16353085/eeed9358-3a10-11e6-99f5-ac8ff4f29859.png">

**JavaScript** with `displayValue: false`:
```js
JsBarcode("#upc-a", "123456789012", {
  displayValue: false,
  format: "upc",
  background: "#000",
  lineColor: "#fff",
  margin: 0
})
```

**Result**:
<img width="265" alt="screen shot 2016-06-24 at 1 38 49 pm" src="https://cloud.githubusercontent.com/assets/8601064/16353094/ffcdbe28-3a10-11e6-80d7-42098393a2f8.png">

**Comment by lindell** [comment]
I will fix this in the beginning of next week :)

**Comment by lindell** [comment]
821922d fixes the problem.

**This:**
````javascript
JsBarcode("#upc-a", "123456789012", {
  displayValue: false,
  format: "upc",
  background: "#000",
  lineColor: "#fff",
  margin: 0
})
````

**Will now generate this:**
![123456789012](http://i.imgur.com/BDL10qm.png)

**Comment by lindell** [comment]
The changes is now included in [v3.3.14](https://github.com/lindell/JsBarcode/releases/tag/v3.3.14) :smile: 

Closing the issue

**Comment by angeloashmore** [comment]
Thank you very much!
